Modules/Forum/classes/class.ilObjForumGUI.php

```diff
@@ -5250,6 +5278,55 @@ private function renderSplitButton(
                     $action_button->setDefaultButton($sb_item);
                     ++$i;
                 } else {
+                    global $DIC;
+                    if ('frm_revoke_censorship' === $lng_id || 'frm_censorship' === $lng_id) {
```

Comment:
I am not sure if the `roundtrip` modal is used as intented in this `censorship` case. Please check the documentation or some other examples (e.g. in `Services/Calendar`).